resources/windows/wix/service.mcollective.wxs.erb

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <DirectoryRef Id="PUPPETBINDIR">
```

Comment:
We do not have this working, However at this point I think it will be better to merge this PR with this in to facilitate other work. Eventually this will either be searched for or defined in the directorylist.wxs file I am working on. EIther way this will stay in. What john was proposing (an idea that I like) is that for now we can merge wix files that can pass the candle step. Then when we have moved past getting light to execute we can make sure all directoryrefs work.